dotnet/src/Functions/Functions.OpenAPI/Extensions/KernelAIPluginExtensions.cs

```diff
@@ -386,6 +386,7 @@ async Task<SKContext> ExecuteAsync(SKContext context)
                 Description = $"{p.Description ?? p.Name}{(p.IsRequired ? " (required)" : string.Empty)}",
```

Comment:
We might want to reconsider this now that we have `IsRequired` -- will also need to update planner manual formatting though to accommodate that. I'm fine not doing that in this PR though. Can you just make sure an issue is filed to track the follow up @gitri-ms when this merges and assign to me?

Reply:
Yes will open an issue